packages/@aws-cdk/aws-synthetics-alpha/package.json

```diff
@@ -117,5 +117,6 @@
       "naming/package-matches-directory",
       "assert/assert-dependency"
     ]
-  }
+  },
+  "deprecated": "This package has been stabilized in aws-cdk-lib is no longer being updated under the -alpha name. Users should migrate to aws-<module> in aws-cdk-lib."
```

Comment:
i'm going to try this. if jsii / npm does the deprecation for us, great. if not, then i will follow up and do it manually